### ISSUE TITLE
Required options missing

### ISSUE THREAD

**Issue by escandethomas** [worldbank/Stata-IE-Visual-Library#50]
https://github.com/worldbank/Stata-IE-Visual-Library/blob/8401d5830aec0d9bfd28ebdf04caf12c720bded1/ado%20files/chartable.sthlp#L13-L14

In the ado file, case0 and case1 are required options, but there is no mention of them in the help file!

**Comment by luizaandrade** [comment]
@bbdaniels, this is one of yours, right? I think the next issue is also about the same command

**Comment by bbdaniels** [comment]
This is a custom ado file that is no longer maintained. It is superseded by https://github.com/bbdaniels/forest which should be up to date. I don't think it is worth it to update things that are now out of date, but they could be pulled from the library and replaced with current commands if you think that's a good solution?

**Comment by luizaandrade** [comment]
Yes, I think so. We'll have a new intern working on this during the summer, so she can do that
